python/mlcroissant/mlcroissant/_src/core/versions.py

```diff
@@ -0,0 +1,34 @@
+"""Versions module."""
+
+from typing import Any
+
+from mlcroissant._src.core.issues import ErrorException
+from mlcroissant._src.core.issues import WarningException
+
+
+def cast_version(version: Any) -> str | None:
+    """Casts the given version and returns a normalized string version.
```

Comment:
NIT: dataset version (to not confuse this with specs version?)

Reply:
Done.